src/components/AppLogin.vue

```diff
@@ -3,11 +3,13 @@
     <input type="text" placeholder="email" v-model="email">
     <input type="password" placeholder="password" v-model="password">
     <button type="submit">Login</button>
-    <div class="spinner" v-if="pending"></div>
+    <div class="spinner" v-if="status"></div>
```

Comment:
Nemoj `v-if="status"`, nejasno je. Ovo gore pending je imalo vise smisla.

Reply:
imam u `AppRegister` `v-if="status === 'loading'"`, ali nisam stavio i ovdje :smile: `v-if="status"` svakako ne radi to sto treba, jer `if(ne_prazan_string)` daje `true`, a meni treba samo ako je status bas === loading :slightly_smiling_face: